src/containers/Payment/index.js

```diff
@@ -35,7 +35,15 @@ const starCount = 5
 class Payment extends Component {
 
     componentDidMount() {
-        this.props.getCheckout()
+        console.log(this.props.match.params)
+        this.props.getCheckout(this.props.match.params.multihash)
+    }
+
+    componentWillReceiveProps(nextProps) {
```

Comment:
Should we use componentDidUpdate instead, as mentioned here: https://reactjs.org/docs/react-component.html#unsafe_componentwillreceiveprops

Reply:
Yes. Great catch